src/v0.1.0/properties/structures/_cheminfo_smiles.yaml

```diff
@@ -0,0 +1,23 @@
+$$schema: "https://schemas.optimade.org/meta/v1.2/optimade/property_definition"
+$id: "https://schemas.optimade.org/namespaces/cheminformatics/v0.1/properties/structures/_cheminfo_smiles"
+title: "SMILES (Simplified Molecular Input Line Entry Specification) representation of the structure"
+x-optimade-type: "string"
+x-optimade-definition:
+  kind: "property"
+  version: "0.1.0"
+  format: "1.2"
+  name: "_cheminfo_smiles"
+  label: "_cheminfo_smiles_structures"
+type:
+  - "string"
+  - "null"
+description: |-
+  SMILES (Simplified Molecular Input Line Entry System) representation of the structure.
+  Values MUST adhere to the OpenSMILES specification v1.0 (http://opensmiles.org/opensmiles.html).
```

Comment:
Consensus is that we should recommend our favourite specifications, rather than necessarily enforcing it.

We can also recommend that implementations announce which flavour of SMILES they are using as human-readable metadata

Reply:
> We can also recommend that implementations announce which flavour of SMILES they are using as human-readable metadata

There was a suggestion of having an enumerator for most-widely-used values and value `other` with another field defined for free-form text.

Reply:
Also just remembered this point that either this field should be multivalued, or we have a proper recommendation for how to deal with multiple disconnected chemical subcomponents, or the case where the SMILES does not include all atoms in the "unit cell"

Reply:
> Also just remembered this point that either this field should be multivalued, or we have a proper recommendation for how to deal with multiple disconnected chemical subcomponents, or the case where the SMILES does not include all atoms in the "unit cell"

I have mentioned this issue in the initial PR message, but please correct me if you meant a different thing.

The issue of not representing all atoms in the structure (sort of "unit cell") is a good catch, though.